spec/requests/movies_spec.rb

```diff
@@ -0,0 +1,87 @@
+require 'rails_helper'
+require 'faker'
+
+RSpec.describe 'Movies', type: :request do
+  describe 'GET /movies' do
+    context 'when there are records' do
+      before { create_list(:movie, 10) }
+
+      it 'shows 10 movies' do
+        get '/movies',
+            params: {}, as: :json
+        expect(JSON(response.body).size > 0).to be(true)
+      end
+    end
+
+    context 'when there is no records' do
+      it 'shows 0 movies' do
+        get '/movies',
+            params: {}, as: :json
+        expect(JSON(response.body).size.equal?(0)).to be(true)
```

Comment:
just check that size eq 0
```suggestion
        expect(JSON(response.body).size).to eq(0)
```